HEC.FDA.Model/structures/Inventory.cs

```diff
@@ -40,8 +41,16 @@ public Inventory(string pointShapefilePath, string impactAreaShapefilePath, Stru
         {
             OccTypes = occTypes;
             PriceIndex = priceIndex;
-            Projection studyProjection = Projection.FromFile(projectionFilePath);//Projection.FromFile returns Null if the path is bad. We'll check for null before we reproject. 
-            Structures = ShapefileLoader.LoadStructuresFromSourceFiles(pointShapefilePath, map, null, false, impactAreaShapefilePath, studyProjection, OccTypes);
+            Projection studyProjection = Projection.FromFile(projectionFilePath);//Projection.FromFile returns Null if the path is bad. We'll check for null before we reproject.
+            OperationResult operationResult = StructureFactory.LoadStructuresFromSourceFiles(pointShapefilePath, map, null, false, impactAreaShapefilePath, studyProjection, OccTypes, out List<Structure> structures);
```

Comment:
Setting up for better messaging by returning an operation result which has messages in it to explain failures.